src/main/kotlin/org/ptss/support/infrastructure/persistence/entities/GroupFamilyMemberEntity.kt

```diff
@@ -0,0 +1,34 @@
+package org.ptss.support.infrastructure.persistence.entities
+
+import io.quarkus.hibernate.orm.panache.kotlin.PanacheCompanion
+import jakarta.persistence.*
+import java.time.OffsetDateTime
+
+@Entity
+@Table(
+    name = "group_family_members",
+    uniqueConstraints = [
+        UniqueConstraint(columnNames = ["user_id"])
+    ]
+)
+class GroupFamilyMemberEntity : BaseEntity() {
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(nullable = false, updatable = false)
+    lateinit var group: GroupEntity
+
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
many to one because of HCP being able to join multiple groups right?

Reply:
Dit gaat alleen om family members dus meerdere family members kunnen één group joinen (N:1)